### DISCUSSION TITLE
Why author opted for python3.9 as package name intead of python

### DISCUSSION THREAD

**Discussion by defencedog** [termux-user-repository/tur#67] [Q&A]
Why author opted for package name like `python3.9` & not just `python`? 
I mean one could have invoked `apt-cache madison python` & choose required version. 
Bcz of this i couldn't use `scipy` ftom its-poinless as it depends on package python & not python3.9

**Comment by licy183** [comment]
I think the packages from tur should not conflict with the packages from main repo, so I can't accept having duplicated package names. If you want to use scipy/numpy from pointless-repo, you could just install `python3.9` before installing `numpy/scipy` and install `openblas=0.3.15` after installing `numpy/scipy`.
BTW, `scipy` is available on `x86_64` and `aarch64`, but not for `arm` and `i686`. I had no idea why this happens, so I disabled these packages (see https://github.com/termux-user-repository/tur/pull/21). They could be added after marking `arm` and `i686` as blacklisted arches. I will submit a commit for it.

**Comment by defencedog** [comment]
You cannot do that; as soon as i install `scipy` it installs `python` package from termux which is `3.10.6` & `scipy` will not run on it. I had installed `python3.9` but how to make `scipy` installation know that `python` already is installed?

**Reply by licy183** [reply]
But you could use `scipy` on `python3.9`, as `scipy` from pointless-repo was compiled for python3.9 and modified its egg list. Of course, you will have both python3.10 and python3.9 on your device.

**Reply by defencedog** [reply]
Ok I will check thanks for replying

**Reply by defencedog** [reply]
Thanks ...Have done this to make life easier
`nano .bash_profile `(in home dir & add following x2 lines)
```
   alias python="python3.9"
   alias pip="python3.9 -m pip"
```
